src/__tests__/select-options.js

```diff
@@ -29,6 +29,53 @@ test('fires correct events', () => {
   expect(o3.selected).toBe(false)
 })
 
+test('fires correct events on listBox select', () => {
+  const {listbox, options, getEventSnapshot} = setupListbox()
+  userEvent.selectOptions(listbox, '2')
+  expect(getEventSnapshot()).toMatchInlineSnapshot(`
+    Events fired on: ul
+
+    ul - pointerover
+    ul - pointerenter
+    ul - mouseover: Left (0)
+    ul - mouseenter: Left (0)
+    ul - pointermove
+    ul - mousemove: Left (0)
+    ul - pointerdown
+    ul - mousedown: Left (0)
+    ul - pointerup
+    ul - mouseup: Left (0)
+    ul - click: Left (0)
+    li[value=0] - pointerover
```

Comment:
Should this be firing on the `li[value=0]` element? I would expect these to fire on `li[value=2]` element since that's the option that's being selection 🤔

Reply:
I will check it, because I think the `li` element has no value just `innerHTML` so this snapshot could be improved

Reply:
Ah yes, I wondered whether it's the utility function acting up here. If you could try to get that fixed that would be great (or you could add an ID to each of the lis).

Reply:
I tried both
- Added ID to each `li`
- Added `[role="listbox"]` and  `[role="option"]` behaviors to `getElementValue` and `getElementDisplayName` utils

What do you think?